app/code/Magento/Review/view/frontend/web/js/process-reviews.js

```diff
@@ -41,7 +41,11 @@ define([
             requiredReviewTabRole = 'tab';
 
         if (reviewTab.attr('role') === requiredReviewTabRole && reviewTab.hasClass('active')) {
-            processReviews(config.productReviewUrl);
+            if (location.hash === '#reviews') {
+                processReviews(config.productReviewUrl, true);
+            } else {
+                processReviews(config.productReviewUrl);
```

Comment:
Can we replace full block with 

```
processReviews(config.productReviewUrl, location.hash === '#reviews');
```

?

Reply:
@orlangur changes applied in https://github.com/magento/magento2/commit/79d323949248c80719def6d3c5c69033ad731ba4